_config.yml

```diff
@@ -1,5 +1,5 @@
 # Website settings
-baseurl: "/"
+baseurl: ""
```

Comment:
What does this change do?

Reply:
From version 3.3, it is recommended in the jekyll docs, to never set the baseurl to "/". I discovered the issue when trying to get the browser reload working. The server address ended up with a double slash at the end like this: `http://0.0.0.0:4000//` when the baseurl is set to "/".

https://jekyllrb.com/news/#2-relative_url-and-absolute_url-filters

Reply:
Understood, thx!